app/assets/javascripts/alchemy/alchemy.initializer.js.coffee

```diff
@@ -46,6 +46,8 @@ Alchemy.Initializer = ->
     tagName = (event.target || event.srcElement).tagName
     key.isPressed('esc') || !(tagName == 'INPUT' || tagName == 'SELECT' || tagName == 'TEXTAREA')
 
+  Alchemy.NodeSorter()
```

Comment:
This only need to run on the `admin/nodes/index.html.erb` template. Especially of you keep the current selector

Reply:
I've changed the selector to be way more specific, so we can keep the JS out of the templates.